snap/snapcraft.yaml

```diff
@@ -52,6 +52,12 @@ parts:
         LIBSODIUM=$(readlink -n $TRIPLET_PATH/libsodium.so.18)
         ln -s $LIBSODIUM $TRIPLET_PATH/libsodium.so
     after: [python, apt]
+  patchelf:
+    source: https://launchpad.net/ubuntu/+archive/primary/+files/patchelf_0.9.orig.tar.gz
```

Comment:
Wouldn't it be better to build the snap with the version that is on xenial, until we release bionic?
Like, keep our snap in sync with the latest LTS, as much as possible.

Reply:
No, because this patchelf needs to be `patchelf`ed itself, additionally, we need the features of this version everywhere for the future PRs related to rpath and no-default-libs

Reply:
Isn't the whole point of this PR that patchelf would still be patchelf'd itself, even if it was a stage-package? More importantly, though, the idea that we'll be relying on v0.9 features when the deb can only ever use v0.8 seems somewhat concerning unless we're quickly moving toward not having a deb at all.

Reply:
not in the first release where it is built from source with the linker flag that it is removed. This is a bootstrapping problem.

We also *DO* need features from 0.9 and I'll be moving to using the snap on builders almost effectively making th deb useless (it already runs behind by a lot given our iterations).

That said, I am making patchelf available as a snap the deb would need to install for everything to work.

Reply:
Alternatively, we could vendor patchelf into snapcraft for the deb.

Reply:
Yeah your call, just something I wanted to point out.